src/OrchardCore.Modules/OrchardCore.Workflows/Services/WorkflowManager.cs

```diff
@@ -16,6 +16,8 @@ namespace OrchardCore.Workflows.Services
 {
     public class WorkflowManager : IWorkflowManager
     {
+        private const int MaxRecursionDepth = 100;
```

Comment:
Should it be configurable? In case someone hits the limit and actually needs to increase it? Or do we think that anything that reaches this point is falty.

Reply:
Yes i thought about it but here it means that a WF1, while starting / resuming in a given scope and before reaching a blocking activity, triggers a WF2 that triggers a WF3, and so on.

So yes i think that reaching 100 is faulty.

I didn't want to complicate things, but no problem if you prefer to have an option

Reply:
Just add a comment there then for later.

Reply:
Okay no problem, i will do it